ember-cli-build.js

```diff
@@ -45,6 +45,12 @@ module.exports = function(defaults) {
   app.import('bower_components/js-polyfills/xhr.js', {
     using: [{ transformation: 'fastbootShim' }]
   });
+  app.import('bower_components/jquery-ui/jquery-ui.js', {
```

Comment:
I don't think travis cache was ever an issue, if you debug properly you will notice that when you visit the create route an error gets logged in the console. 
the line involves the form.` fn.$.settings..` this $ is being overwritten due to these imports, solve it locally first, then test on travis, if the tests fail locally, travis cahce  won't matter at all.

Reply:
Thanks for pointing that out. But I guess Travis caching was also an error because it was caching a previous version of the library which did not include the `jquery-ui.js` file. This was happening locally as well. That's why I earlier imported from `node_modules`, which is unconventional. Anyways, that didn't resolve the main issue, so working on it :) 👍

Reply:
Seems like `jquery` and `jquery-ui` libraries are in conflict.

Reply:
I tried using Jquery’s $.noConflict() method to fix this, but it seems to have no effect.